src/hub.js

```diff
@@ -334,7 +333,7 @@ const onReady = async () => {
       document.body.addEventListener("connected", () => {
         if (!isBotMode) {
           hubChannel.sendEntryEvent().then(() => {
-            store.update({ activity: { lastEnteredAt: moment().toJSON() } });
```

Comment:
unfortunately we can't do this, because this is a persistent schema in local storage. we'll need to either create a new field or write something to migrate the old data. (new clients will throw when the json is passed to the Date constructor I presume)

Reply:
(my preference is we write a migration script given that this will break our stats otherwise)

Reply:
ah I misunderstood. i assumed it was returning some kind of JSON blob